src/test/java/Homework18.java

```diff
@@ -0,0 +1,33 @@
+import org.apache.hc.core5.util.Asserts;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import java.lang.Thread;
+
+public class Homework18 extends BaseTest {
+
+    @Test
+    public void playSong() throws InterruptedException {
+        launchBrowser();
```

Comment:
When you extend BaseTest and this BaseTest class already exists `launchBrowser()` in @BeforeSuite. So when run any suite `launchBrowser ` method is always called to lanch browser. so you don't need to call `launchBrowser(); `here again. It will open 2 browsers which will cause error during testing. Please romove it.